<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModifyUser</class>
 <widget class="QDialog" name="ModifyUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modify User</string>
  </property>
  <widget class="QPushButton" name="btnModify">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Modify</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnReturn">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblMessage">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 0)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="leUsername">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>60</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblUser">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>68</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>User</string>
   </property>
   <property name="buddy">
    <cstring>leUsername</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="lblPassword">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>New Password</string>
   </property>
   <property name="buddy">
    <cstring>lePassword</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lePassword">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>leUsername</tabstop>
  <tabstop>lePassword</tabstop>
  <tabstop>btnReturn</tabstop>
  <tabstop>btnModify</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnReturn</sender>
   <signal>clicked()</signal>
   <receiver>ModifyUser</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>220</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>239</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnModify</sender>
   <signal>clicked()</signal>
   <receiver>ModifyUser</receiver>
   <slot>onModifyUser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>265</x>
     <y>217</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lePassword</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ModifyUser</receiver>
   <slot>onEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leUsername</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ModifyUser</receiver>
   <slot>onEdit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onModifyUser()</slot>
  <slot>onEdit()</slot>
 </slots>
</ui>
